<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>755</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <strikeout>false</strikeout>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Eye-Tracking Visualizer</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1065</width>
      <height>753</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="_plotting_row">
     <item>
      <layout class="QVBoxLayout" name="_plotting_row_options">
       <item alignment="Qt::AlignHCenter">
        <widget class="QScrollArea" name="_error_message_scroll_area">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>325</width>
           <height>46</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>325</width>
           <height>46</height>
          </size>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_5">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>323</width>
            <height>44</height>
           </rect>
          </property>
          <widget class="QLabel" name="_error_message">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>325</width>
             <height>46</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>325</width>
             <height>46</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>325</width>
             <height>46</height>
            </size>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="_directory_selection_button">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>Select Directory</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="_vertical_spacer_directory">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="_select_all_button_box">
         <item>
          <widget class="QPushButton" name="_participant_select_all_button">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>21</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>21</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Select All</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="_horizontal_spacer_select_deselect_all">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="_participant_deselect_all_button">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>82</width>
             <height>21</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>82</width>
             <height>21</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Deselect All</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QScrollArea" name="_participant_selection_menu">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>325</width>
           <height>350</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>325</width>
           <height>350</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="_participant_selection_widget_holder">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>323</width>
            <height>348</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="_participant_selection_button">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>Select Participant(s)</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="_stimulus_horizontal_layout">
         <item>
          <widget class="QLabel" name="_stimulus_label">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Stimulus:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="_stimulus_selection_menu">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>23</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="_data_type_horizontal_layout">
         <item>
          <widget class="QLabel" name="_data_type_label">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Data Type:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="_data_type_selection_menu">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>23</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="_analysis_type_horizontal_menu">
         <item>
          <widget class="QLabel" name="_analysis_type_label">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Analysis Type:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="_analysis_type_selection_menu">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>23</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="_axes_selection_check_box">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>23</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Show Axes</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="_only_data_on_stimulus_selection_check_box">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>Only Show Data on Stimulus</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="_plot_button">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>Plot</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="_vertical_spacer_bottom">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="_plotting_row_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>1</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="_plotting_row_plot">
       <item>
        <widget class="QWidget" name="_plot_placeholder" native="true">
         <property name="minimumSize">
          <size>
           <width>700</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>700</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
